addons/cba_settings_userconfig/config.cpp

```diff
@@ -0,0 +1,14 @@
+
+class CfgPatches {
+    class cba_settings_userconfig {
+        author = "$STR_CBA_Author";
+        name = "$STR_CBA_Settings_Component";
+        url = "$STR_CBA_URL";
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = 1.0;
+        requiredAddons[] = {"cba_settings"};
+        version = 1.0;
+        authors[] = {"commy2"};
```

Comment:
:thinking:

Reply:
it's a template, see no reason to change this.